DEVELOPMENT.md

```diff
@@ -26,6 +26,25 @@ To run end-to-end tests you will also need:
 
 Installation instructions for these tools differ heavily based on your operating system and aren't covered here.
 
+### CouchDB on Docker
+
+We recommend using Docker to install and use CouchDB. This ensures you are getting a compatible version and are not relying on OS packages that may be more or less up to date than needed. 
+
+After installing docker, you can create a docker container like so:
```

Comment:
Maybe make `installing docker` a link to [docker's docs](https://docs.docker.com/get-docker/)?

Reply:
Nice.